mobile/views/Search/PeopleSearchView.js

```diff
@@ -19,8 +19,9 @@ type Props = {
     search: SearchUsersType,
   },
   ...$Exact<ViewNetworkHandlerProps>,
-  navigation: NavigationProps,
+  onPress: (userId: string) => any,
```

Comment:
@brianlovin this is the most interesting change code-wise. Instead of automatically redirecting the to the user profile when clicking on a user search result I added an onPress prop, which means I can fully reuse this existing component in the dm composer! 😍😍😍 (I made it go to the user profile on the search view, obviously)